src/pages/solutions/for-students.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: We help students 
+quickLinks:
+  - name: Data
+    anchor: data
+  - name: Basemaps and Imagery
+    anchor: imagery
+  - name: Conferences and Events
+    anchor: conferences
+  - name: Jobs and Internships
+    anchor: jobs
+  - name: Metadata
+    anchor: metadata
+  - name: Citations
+    anchor: citations
+
+---
+
+import Layout from '../../layouts/FullWidth.astro';
+import Section from '../../components/page/Section.astro';
+import QuickLinks from '../../components/page/QuickLinks.astro';
+
+<Layout title={frontmatter.title}>
+  <QuickLinks links={frontmatter.quickLinks}/>
+  <Section stripe title="Data">
```

Comment:
It's not in the government template yet, but I would like to suggest that we have another section without a title or with the For Students title that introduces the solutions below. 

e.g. data is important for school projects. we have the best data for your projects to get an A. Are you almot done with school and want to work in our industry? We can help you there also. Make something cool? Tweet us about it, we'd love to hear.